### PR TITLE
docs: Update 1.117 release notes

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by justinsb** [comment]
/assign @jingyih

In case you have opinions about how we should describe the VertexAI changes in release notes!

File: docs/releasenotes/release-1.117.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# v1.116.0
```
Comment (jingyih) [inline docs/releasenotes/release-1.117.md:1]: 1.117?

File: docs/releasenotes/release-1.117.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* ...
+
+* Special shout-outs to ... for their
+  contributions to this release.
+TODO: list contributors with `git log v1.114.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+
+## Resources promoted from alpha to beta:
+
+*When resources are promoted from alpha to beta, we (generally) ensure they follow our best practices: use of refs on fields where appropriate,
+output fields from GCP APIs are in `status.observedState`.*
```
Comment (jingyih) [inline docs/releasenotes/release-1.117.md:14]: nit: `status.observedState.*`?

**Pull request merged**